### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by edidee**

### CODE REVIEW

**Comment by edidee** [comment]
meStyle

File: app/components/landingpage/Hero.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    <section className="w-full h-screen p-8 space-y-3 bg-black text-white text-center">
+      <div className="space-y-3 mt-4">
+        <p className="md:text-5xl text-2xl font-lexend ">
+          Get Your Confidence Again
+        </p>
+        <p className="md:text-2xl text-[10px]">
+          Discover the latest trends and showcase your unique style
+        </p>
+
+        <button className="p-3 bg-white text-black" type="button">
```
Comment (Remi-dee) [inline app/components/landingpage/Hero.js]: Kindly import the reusable button from the `ui` directory and use that instead

**Review by Remi-dee — CHANGES_REQUESTED** [review]

File: app/components/landingpage/Hero.js (hunk 1 above)
Comment (edidee) [reply app/components/landingpage/Hero.js]: Done

**Review by Remi-dee — APPROVED** [review]
LG

**Pull request merged**